### PR TITLE
Port PLN Plugin to 3.3

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

File: classes/DepositObjectDAO.inc.php [hunk 1]
```diff
@@ -113,26 +106,23 @@ public function markHavingUpdatedContent($journalId, $objectType) {
...
 					['issueId', STATUS_PUBLISHED, (int) $journalId]
 				);
-				while (!$result->EOF) {
-					$row = $result->GetRowAssoc(false);
-					$depositObject = $this->getById($journalId, $row['deposit_object_id']);
+				foreach ($result as $row) {
+					$depositObject = $this->getById($journalId, $row->deposit_object_id);
 					$deposit = $depositDao->getById($depositObject->getDepositId());
-					if($deposit->getSentStatus() || !$deposit->getTransferredStatus()) {
+					//if ($deposit->getSentStatus() || !$deposit->getTransferredStatus()) {
```
Comment (asmecher) [inline classes/DepositObjectDAO.inc.php]: Commented code; is this intentional?

Comment (defstat) [reply classes/DepositObjectDAO.inc.php]: Yes. It's to compare the before state and after state when reviewed in fact.

I think that we should repackage and send the deposit to the staging server if it has new content, no matter what it's current state. But I needed your opinion on that.

Comment (defstat) [reply classes/DepositObjectDAO.inc.php]: I think we can return the code to the previous state for that (as it is not related to 3.3 port) and consider that along with the other state related changes.

Comment (asmecher) [reply classes/DepositObjectDAO.inc.php]: Ah, OK, thanks!

Comment (defstat) [reply classes/DepositObjectDAO.inc.php]: @asmecher OK I removed the change

Comment (defstat) [reply classes/DepositObjectDAO.inc.php]: I will merge it and create a release based on the master branch.

**Pull request merged**